### PR TITLE
feat: VSDSPUB-192: fix broken SHACL validation

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java [hunk 1]
```diff
@@ -2,18 +2,26 @@
...
 
-	private final MemberIngestService memberIngestService;
+	@Autowired
+	MemberIngestService memberIngestService;
+	@Autowired
+	private LdesShaclValidator validator;
 
-	public LdesMemberIngestionController(final MemberIngestService memberIngestService) {
-		this.memberIngestService = memberIngestService;
+	@InitBinder
```
Comment (Tomvbe) [inline ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java:22]: Oei waarom is dit nodig?

File: ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java [hunk 2]
```diff
@@ -2,18 +2,26 @@
...
+import org.springframework.web.bind.annotation.InitBinder;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class LdesMemberIngestionController {
 
-	private final MemberIngestService memberIngestService;
+	@Autowired
```
Comment (Tomvbe) [inline ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java:17]: Can't we keep ctr injection?

cfr first answer here for some good arguments:
https://stackoverflow.com/questions/40620000/spring-autowire-on-properties-vs-constructor

File: ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java (hunk 1 above)
Comment (Yalz) [reply ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java:22]: To use  @Validated in the rest controller, this validator needs to be binded. therefor the initbind

File: ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/LdesShaclValidator.java [hunk 3]
```diff
@@ -33,16 +31,21 @@ public boolean supports(Class<?> clazz) {
...
 	public void validate(Object target, Errors errors) {
 		Member member = (Member) target;
-		validateShape(member, errors);
+		validateShape(member);
 	}
 
-	protected void validateShape(Member member, Errors errors) {
+	protected void validateShape(Member member) {
+		if (!initialized) {
+			if (validationConfig.isEnabled() && validationConfig.getShape() != null) {
```
Comment (Tomvbe) [inline ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/LdesShaclValidator.java:39]: can't you create a 'shapes' bean in a configuration class and just inject it in this validator?

**Review by Tomvbe — APPROVED** [review]

**Pull request merged**